modules/configuring-the-codeflare-operator.adoc

```diff
@@ -0,0 +1,102 @@
+:_module-type: PROCEDURE
+
+[id="configuring-the-codeflare-operator_{context}"]
+= Configuring the CodeFlare Operator
+
+[role='_abstract']
+If you want to change the default configuration of the CodeFlare Operator for distributed workloads in {productname-short}, you can edit the associated config map.
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
```

Comment:
You have installed the CodeFare Operator.

Reply:
You don't need to install the CodeFlare Operator separately, it's included in RHOAI and ODH.